### PR TITLE
stale-while-revalidate blog post

### LINKED ISSUE #1000: Stale While Revalidate article [web.dev/blog]
**A one to two sentence description of your post**

Stale-while-revalidate (SWR) is used by the Cache-Control header. It defines an extra window of time during which a cache can use a stale asset while it revalidates it asynchronously. This improves subsequent page load latencies as stale assets are non longer in the critical path.

We would like to author an article on SWR as it's shipping in Chrome. @jeffposnick has kindly offered to author this and is in touch with Kenji and Dave Tapuska on implementation questions.

**Target publish date:** `<yyyy-mm-dd>`

- [ ] Check this box if this is a hard deadline.

**Process**

Take a moment to look over the [web.dev handbook](https://web.dev/handbook) and
familiarize yourself with the process.

When you're ready, make a copy of [this template](https://docs.google.com/document/d/1ydauCufwwavStaKxhIDHuNivVxgQBOdDno4uLjGIjmE/edit)
and fill in the proposal and outline sections. When you're finished, drop a link
to the doc in the 1st step below and check the box. A member of the web.dev team
will assign this ticket to themselves and work as your reviewer to help you
complete the remaining steps.

- [x] 1. I've done my proposal `<add a link to your proposal doc>`
- [x] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a draft to my proposal doc. https://docs.google.com/document/d/1f55OIGCqevtWpuL04RuIGGlv981Ew5AVa0A_6Tdd33Q/edit
- [x] 4. Draft approved — _reviewer checks this_
- [x] 5. I've submitted a pull request #1140 
- [ ] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by jeffposnick**
Fixes #1000 

Changes proposed in this pull request:

- A blog post explaining how to use the newly implemented `stale-while-revalidate` option in the `Cache-Control` header.

The content has been already reviewed by @jpmedley.

### CODE REVIEW

File: src/site/content/en/blog/stale-while-revalidate/index.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+request. The browser will instead retrieve a response from the network, and use
+that for both fulfilling the initial request and also populating the local cache
+with a fresh response.
+
+# Live Example
+
+Below is a simple example of an HTTP API for returning the current time—more
+specifically, the current number of minutes past the hour.
+
+<div class="glitch-embed-wrap" style="height: 346px; width: 100%;">
```
Comment (jeffposnick) [inline src/site/content/en/blog/stale-while-revalidate/index.md]: Let me know if there's a newer/better way of embedding a Glitch...

File: src/site/content/en/blog/stale-while-revalidate/index.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  - jeffposnick
+date: 2019-07-15
+description: stale-while-revalidate helps developers balance between immediacy—loading cached content right away—and freshness—ensuring updates to the cached content are used in the future.
+tags:
+  - post
+  - speed
+  - caching
+---
+
+# What shipped?
```
Comment (robdodson) [inline src/site/content/en/blog/stale-while-revalidate/index.md]: Change all headings to `##`

File: src/site/content/en/blog/stale-while-revalidate/index.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+used within a [service worker](/service-workers-cache-storage/).
+
+Using stale-while-revalidate via a `Cache-Control` header shares some
+similarities with its use in a service worker, and many of the same
+considerations around freshness trade-offs and maximum lifetimes apply. However,
+there are a few considerations that you should take into account when deciding
+whether to implement a service worker-based approach, or just rely on the
+`Cache-Control` header configuration.
+
+**Use a service worker approach if...**
```
Comment (robdodson) [inline src/site/content/en/blog/stale-while-revalidate/index.md]: suggest making these h3s

**Comment by jeffposnick** [comment]
I've:

- added a hero image
- switch the inline PNG to SVG
- updated the date
- changed the headings

**Pull request merged**